packages/next-server/server/next-server.ts

```diff
@@ -312,13 +312,16 @@ export default class Server {
         return mod.default(req, res)
       }
     }
-
-    apiResolver(
-      req,
-      res,
-      params,
-      resolverFunction ? require(resolverFunction) : undefined
-    )
+    try {
+      await apiResolver(
+        req,
+        res,
+        params,
+        resolverFunction ? require(resolverFunction) : undefined
+      )
+    } catch (e) {
```

Comment:
The `await` above will automatically throw the error -- we don't need this `try { ... } catch (...) { ... }`. Please remove it.